src/python/grpcio_tests/tests/qps/qps_worker.py

```diff
@@ -39,8 +39,13 @@ def run_worker_server(port):
         description='gRPC Python performance testing worker')
     parser.add_argument('--driver_port',
                         type=int,
-                        dest='port',
+                        dest='driver_port',
                         help='The port the worker should listen on')
+    parser.add_argument('--server_port',
+                        type=int,
+                        default=0,
```

Comment:
It would be more conventional to use `None` as the default value, here and all the way down the stack. Especially since 0 is a valid (and potentially desirable) port to bind to.

Reply:
Fixed to None.